macros/incremental_days_forward_predicate.sql

```diff
@@ -0,0 +1,3 @@
+{% macro incremental_days_forward_predicate(column, base_time, days_forward, interval_period) -%}
+        {{column}} BETWEEN date_trunc('day', base_time - interval '{{var('DBT_ENV_INCREMENTAL_TIME')}}' day) AND date_trunc('day', base_time + interval '{{days_forward}}' day)
```

Comment:
Do we think the env var will always be `day`?

I overwrote it here, since the usage_summary macro is daily, I'd want to avoid overwriting old full day data with a partial day.

Reply:
i have a hard time imagining a scenario where we'd switch off day. you could also pass from the model to the macro the same variable, and "override" with day again? that may be overkill. i think it's safe to assume we wouldn't change that off day.

Reply:
making the env var, not adding in an override option for now.

Reply:
nevermind, gpt came up with an easy way to pass it in